### PR TITLE
tfel: update to 3.4.0

### PR DESCRIPTION
**Pull request by 3rav**

### CODE REVIEW

File: mingw-w64-tfel/PKGBUILD [hunk 1]
```diff
@@ -15,17 +15,13 @@ makedepends=("unzip"
...
-            '7E85A1F5BAE743C5DAF019051BF743C25CDC84F4BB5D97E22278D527E816B23A')
+source=(https://github.com/thelfer/${_realname}/archive/TFEL-${pkgver}.zip)
+sha256sums=('603FE64C2FD1584F233B08200C57F65D1214EB741B85FD636172DA3E369467E5')
 noextract=("TFEL-${pkgver}.zip") # bsdtar could not unpack this archive
 
 prepare() {  
   rm -rf "${srcdir}/TFEL-${pkgver}r"
   unzip -oq TFEL-${pkgver}.zip -d ${srcdir}
-  cd "${srcdir}/${_realname}-TFEL-${pkgver}"
-  patch -p1 -i "${srcdir}/0000-fix-gcc-10.patch"
```
Comment (lazka) [inline mingw-w64-tfel/PKGBUILD:28]: You forgot to delete the file

File: mingw-w64-tfel/PKGBUILD [hunk 2]
```diff
@@ -34,10 +30,13 @@ build() {
 
   MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
     ${MINGW_PREFIX}/bin/cmake \
-      -G'Ninja' \
+      -GNinja \
       -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
       -DCMAKE_BUILD_TYPE=Release \
+      -DTFEL_APPEND_VERSION=ON \
```
Comment (lazka) [inline mingw-w64-tfel/PKGBUILD:36]: That changes the DLL name and breaks existing users. Why is this needed?

File: mingw-w64-tfel/PKGBUILD [hunk 3]
```diff
@@ -34,10 +30,13 @@ build() {
...
       -Denable-fortran=ON \
+      -Denable-python=OFF \
+      -Denable-python-bindings=OFF \
```
Comment (lazka) [inline mingw-w64-tfel/PKGBUILD:39]: Just wondering, why?

File: mingw-w64-tfel/PKGBUILD [hunk 4]
```diff
@@ -50,13 +49,14 @@ build() {
       -Denable-lsdyna=ON \
       -Denable-cxx-17=ON \
       -Denable-portable-build=ON \
+      -Denable-reference-doc=OFF \
```
Comment (lazka) [inline mingw-w64-tfel/PKGBUILD:52]: Why?

File: mingw-w64-tfel/PKGBUILD (hunk 2 above)
Comment (3rav) [reply mingw-w64-tfel/PKGBUILD:36]: Because: https://github.com/msys2/MINGW-packages/issues/7220
and
https://sourceforge.net/p/tfel/tickets/251/#c29d

File: mingw-w64-tfel/PKGBUILD (hunk 3 above)
Comment (3rav) [reply mingw-w64-tfel/PKGBUILD:39]: I plan to introduce python support but I currently have a bug - error, I think it will be fixed in 3.4.1
https://sourceforge.net/p/tfel/tickets/251/#49c5/783c

File: mingw-w64-tfel/PKGBUILD (hunk 1 above)
Comment (3rav) [reply mingw-w64-tfel/PKGBUILD:28]: Yes, sorry

File: mingw-w64-tfel/PKGBUILD (hunk 4 above)
Comment (3rav) [reply mingw-w64-tfel/PKGBUILD:52]: Reducing the number of files

File: mingw-w64-tfel/PKGBUILD (hunk 2 above)
Comment (lazka) [reply mingw-w64-tfel/PKGBUILD:36]: But adding a version isn't proper namespacing. I'd try to decide which package should ship these tools and remove them from all others.

File: mingw-w64-tfel/PKGBUILD (hunk 3 above)
Comment (lazka) [reply mingw-w64-tfel/PKGBUILD:39]: ok, thansk

File: mingw-w64-tfel/PKGBUILD (hunk 4 above)
Comment (lazka) [reply mingw-w64-tfel/PKGBUILD:52]: fair enough.

File: mingw-w64-tfel/PKGBUILD (hunk 2 above)
Comment (lazka) [reply mingw-w64-tfel/PKGBUILD:36]: Please revert this change

**Comment by lazka** [comment]
revert the naming change and delete the file, then this is good to go.
